crates/services/gas_price_service/src/fuel_gas_price_updater.rs

```diff
@@ -113,16 +117,22 @@ pub trait L2BlockSource: Send + Sync {
     async fn get_l2_block(&mut self, height: BlockHeight) -> Result<BlockInfo>;
 }
 
-#[derive(Debug, Default)]
-pub struct DaCommitDetails {
+#[derive(Debug, Default, Clone)]
+pub struct DaGasPrice {
```

Comment:
I think this should be named something like `DaBlockCosts` or something like that. We use this value to calculate the "Price" but it isn't the price yet.

Reply:
addressed in https://github.com/FuelLabs/fuel-core/pull/2163/commits/74fc9bf4e08cb0d4a1a5def23f31d2c4fbd31c9f#diff-f8482408ef6bf8c1120f3a3a5844ee9cdc5281443f3eea46611822ababd3c7a9R121